### PR TITLE
Hello_Microverse-_Project#1

### PR DESCRIPTION
**Pull request by Kel-nana**
its a simple html and css project

### CODE REVIEW

File: .github/workflows/linters.yml [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+     
+      - uses: actions/checkout@v3
+      - name: Super-Linter
+        uses: github/super-linter@v4.10.0
+
```
Comment (IngDavidTM) [inline .github/workflows/linters.yml:27]: - [ ] You did an amazing job configuring linters :+1: on your repo but as a general requirement for the project you shouldn't custom changes to the linter config and also you shouldn't add linters.yml twice on your repo. If you are having problems with the linters configuration you can always read more [here](https://github.com/microverseinc/linters-config/tree/master/html-css)

**Review by IngDavidTM — CHANGES_REQUESTED** [review]
## Hi @Kel-nana  🙋‍♂️, 

Good job so far!
You have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!



## Top highlight 🎯
- The Pull Request follows the GitHub Flow ✅ 
- Linters are working properly ✅ 

## Suggested changes

- [ ] Kindly don't forget that Pull Request should have a descriptive title telling you what changes are inside, if you have doubts I recommend you to check [Professional repo - basic rules](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md) :+1: 

- [ ] You did a good job implementing the README file :+1: but as a [project requirement](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/hello_microverse_project.md) you should customize the file because it would make your project looks more professional and this is one of the first things that hiring managers or your potential colleagues will check when considering you for a position on their team. To learn more you could read [Document your repo in a professional way](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/clean_repo.md)
[Here](https://github.com/microverseinc/readme-template) you can find the template that you can follow.

- [ ] As a [project requirement](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/hello_microverse_project.md) you should add two more files to the project, `html` file and `css` file to displays 'hello microverse!' with your own style :+1:  


- [x] _Check the comments under the review._ ⬇

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by AHP15 — CHANGES_REQUESTED** [review]
Hi @Kel-nana , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.




 ## Highlights
✅ Good commit messages
✅ Good GitHub flow

 ## Required Changes ♻️

 - [  ] White you did an amazing job adding the changes required by the previous reviewer, but you added them in a new branch (`htmlcss-branch`). Kindly put those changes in this same branch (`super-linter`) because it is included in the requirement:
 
InkedPR_


for more info please visit the [code review article](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/code-review/articles/how_to_ask_for_a_code_review.md)

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code. Some of them were missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by victorokoroji — APPROVED** [review]
Hi @Kel-nana, 

# ✔️   Status: APPROVED 🏆 🎉 🎊 

You have done a great job on this project, keep it up 👍 🥇. Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
congratulations!  🎉💯🌟 

## Top highlight 🎯

- The Pull Request is following the GitHub Flow ✅ 
- The PR title and description are professional and descriptive ✅ 
- Linters are working properly ✅ 


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Kel-nana** [comment]
interesting experience

**Pull request merged**